doc/whats_new/v1.6.rst

```diff
@@ -61,7 +61,9 @@ See :ref:`array_api` for more details.
 - :func:`sklearn.metrics.pairwise.cosine_distances` :pr:`29265` by :user:`Emily Chen <EmilyXinyi>`;
 - :func:`sklearn.metrics.pairwise.cosine_similarity` :pr:`29014` by :user:`Edoardo Abati <EdAbati>`;
 - :func:`sklearn.metrics.pairwise.euclidean_distances` :pr:`29433` by :user:`Omar Salman <OmarManzoor>`;
+- :func:`sklearn.metrics.pairwise.laplacian_kernel` :pr:`29881` by :user:`Omar Salman <OmarManzoor>`;
 - :func:`sklearn.metrics.pairwise.linear_kernel` :pr:`29475` by :user:`Omar Salman <OmarManzoor>`;
+- :func:`sklearn.metrics.pairwise.manhattan_distances` :pr:`29881` by :user:`Omar Salman <OmarManzoor>`;
```

Comment:
I think we should have a dedicated list of functions and classes that accept array API inputs only for convenience but actually convert to NumPy to perform the computation on the host CPU without delegating to the input's namespace.